Appsilon/shiny.benchmark
# Adding inst/examples and a function to create examples structure

### Link to the Issue

close #70 

### Definition of Done

User should be able to create an fake example to test our package.

### How to test changes

Install the package properly using the `install.packages` function for example. Then use `shiny.benchmark::load_example(path = "path_to_new_project")`. It will create an app in `path_to_new_project` and some instructions will appear inside `run_tests.R` file.

### Tasks for PR author


### General policy:

- If applicable - add instructions for testing
- If there’s no issue, create it. Each issue needs to be well defined and described.
- All interaction with a user, user-facing messages, plots, reports etc. are written from the perspective of the person using or receiving it. They are understandable and helpful to this person. If a user sees an error message, there is a call to action, i.e. the user knows what to do to fix it.
- README, other documentation and code comments that we have is updated with all information related to the change.
- All code has been peer-reviewed before merging into any main branch
- All changes have been merged into the main branch we use for development.
- Continuous integration checks (linter, unit tests, integration tests) are configured and pass.
- Optional: unit tests added for all new or changed logic.
- All task requirements satisfied. If not describe it here. The reviewer is responsible to verify each aspect of the task.
- Change covers only things in task. Please create new PR if you want to fix something else.
